public/scripts/stackedbar-util.js

```diff
@@ -1,15 +1,83 @@
 ////////////////////////////// For running time stacked bar ///////////////////
-var StackedBarUtil = (function($) {
-  var sbutil = {};
+var StackedBarUtil = (function($, StateUtil) {
```

Comment:
The names are horrible for all those, it would be nice to sit down and rename all the different chart utility classes in some coherent way soon. I'll think about it but if you think of anything you like let me know!